src/server.ts

```diff
@@ -467,6 +475,48 @@ export class Server {
       transaction.toXDR(),
     );
   }
+
+  /**
+   * Use the friendbot faucet to create and fund a new account. If friendbot is
+   * not configured on this network, this method will throw an error. The
+   * method will return an Account object for the created account. If the
+   * request fails, this method will throw an error.
```

Comment:
also reminder to update the docstring for the new "retrieve on error" behavior

Reply:
How is that different than what this says?

Reply:
I misread it somehow; you're right :+1: maybe we can say `"...for the created account, returning it if it already existed."